docs/api-docs/partner/Resources

```diff
@@ -0,0 +1,22 @@
+# Resources
+
+<div class="otp" id="no-index">
+
+### On this page
+- [API clients](#API-clients)
+- [App starters](#App-starters)
+- [BigDesign](#BigDesign)
+- [SDK (Checkout SDK)](#SDK)
+</div> 
+
+## API clients
+Accelerate your app development with BigCommerce API clients. The API clients are available in different programming languages such as Python, Php, Ruby, and JavaScript. See [Tools & Resources](https://developer.bigcommerce.com/tools-resources) for a list of our API clients.
+
+## App starters
+There are apps that every BigCommerce Enterprise customer should have to enhance their customer experience and sell more online. See [The BigCommerce Enterprise App Starter Kit](https://www.groovecommerce.com/ecommerce-blog/bigcommerce-enterprise-app-starter-kit/) for a list of the must-have apps for all store owners. 
```

Comment:
I am hesitant to link out to external resources. Confirmed with Corey that we can link to sample apps (see Hello World Apps in [Tools](https://developer.bigcommerce.com/tools-resources)). 
```suggestion
BigCommerce sample apps are open-source projects developed by the BigCommerce team and community. For inspiration and to see the code behind each project, visit each individual repository.

- [Python App](https://github.com/bigcommerce/hello-world-app-python-flask)
- [PHP App](https://github.com/bigcommerce/hello-world-app-php-silex)
- [Ruby App](https://github.com/bigcommerce/hello-world-app-ruby-sinatra)
- [OmniAuth Bigcommerce Strategy](https://github.com/bigcommerce/omniauth-bigcommerce)
-[AWS Amplify App](https://github.com/hatertron3000/amplify-bigcommerce)

```